style.css

```diff
@@ -0,0 +1,557 @@
+.caption-title {
+    font-weight: 700;
+    padding: 3%;
+}
+
+.caption-title-header,
+.tech-most-popular-header {
+    color: #fff;
+    margin-bottom: 1.5%;
+    margin-top: 1.5%;
+    padding-top: 0;
+    position: relative;
+    top: 70%;
+}
+
+.caption-title p,
+.tech-most-popular p {
+    color: #fff;
+    margin: 0;
+    position: relative;
+    top: 70%;
+}
+
+.tech-most-popular-header:hover,
+.caption-title-header:hover {
+    color: white;
+    text-decoration: underline solid #999;
+}
+
+.tech-most-popular p a:hover {
+    color: white;
+}
+
+.tech-news {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+}
+
+.tech-news section {
+    color: #555;
+    flex: 3;
+    margin: 0 1%;
+    min-width: 300px;
+}
+
+.tech-news-block {
+    background: rgb(255, 68, 34);
+    background: rgba(255, 68, 34, 0.06);
+    margin: 2% 0;
+}
+
+.tech-news-block li::before {
+    color: #f42;
+    content: "•";
+    margin-left: -1em;
+    margin-right: 0.6em;
+}
+
+.tech-news-block li {
+    background-color: #fff;
+    border-bottom: 1px solid #eee;
+    color: #555;
+    font-size: 13px;
+    list-style: none;
+    margin-left: 0.3em;
+    padding: 1% 0 1% 2%;
+}
+
+.tech-news-block ul {
+    background-color: #fff;
+    padding: 0;
+    padding-left: 1%;
+}
+
+.tech-news-block ul li a:hover {
+    color: rgb(43, 42, 42);
+}
+
+h5:hover {
+    text-decoration: underline;
+}
+
+.tech-latest {
+    color: #f42;
+    padding-left: 4%;
+}
+
+.figure-tesla {
+    background: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(195, 55, 100, 0.7)), url(./resources/tesla-img.jpg);
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    height: 100%;
+    margin: 0 1.5%;
+}
+
+.figure-tesla:hover {
+    opacity: 0.9;
+}
+
+.twd-img {
+    margin: 3%;
+    width: 40%;
+}
+
+.figure-date {
+    color: #fff;
+    float: right;
+    font-size: 80%;
+    padding-right: 1.5%;
+}
+
+.header-column2 figure {
+    margin-left: 0;
+    margin-right: 3%;
+    margin-top: 0;
+    overflow: hidden;
+}
+
+.figure-aside, .figure-aside2,
+.tech-most-popular {
+    background-image: linear-gradient(to top, #333, #555 30%, transparent), url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/01/trump-tesla-grant-money-inject-research-hydrogen-science-796x448.jpg);
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    color: #000;
+}
+
+.figure-aside, .figure-aside2 {
+    margin-bottom: 0;
+    padding-top: 25%;
+}
+
+.figure-aside {
+    margin-bottom: 3%;
+}
+
+.figure-aside:hover, .figure-aside2:hover {
+    opacity: 0.9;
+}
+
+footer {
+    background-color: #131313;
+    color: #555;
+    padding-top: 20px;
+    text-align: center;
+}
+
+.copyright {
+    background: #000;
+    color: #aaa;
+    font-size: 75%;
+    padding: 3%;
+    text-align: center;
+}
+
+
+button {
+    background: #333;
+    border-color: #333;
+    border-radius: 50%;
+    color: #fff;
+    font-size: large;
+    height: 40px;
+    width: 40px;
```

Comment:
You could remove the default border from the buttons (border: none) and may be set here the line-height: 1.